### PR TITLE
chore: more readme, node 12, replace greenkeeper badge etc, #noissue

### PR DESCRIPTION
**Pull request by sadams**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -44,10 +41,8 @@ jobs:
         executor: default
         steps:
             - *attach_workspace
-            # - checkout # Commenting this line out as I don't think it is needed, please uncomment if the release fails
```
Comment (sadams) [inline .circleci/config.yml:47]: @MrSwitch don't know if this isn't needed yet. waiting to see if https://github.com/5app/region-sync publishing works once the PR is approved.

File: package.json [hunk 2]
```diff
@@ -8,9 +8,9 @@
 		"lint": "eslint ./",
 		"lint-fix": "npm run lint -- --fix",
 		"lint-diff": "LIST=`git diff-index --name-only HEAD | grep '.*\\.js$';`; if [ \"$LIST\" ]; then eslint $LIST; fi",
-		"precommit-msg": "echo 'Running pre-commit checks... (skip using --no-verify)' && exit 0",
+		"precommit-msg": "echo 'Running pre-commit checks... (skip using --no-verify)'",
 		"semantic-release": "semantic-release",
-		"test": "echo 'No tests' && exit 0"
+		"test": "echo 'implement tests' && exit 1"
```
Comment (ikhemissi) [inline package.json:13]: 👍

File: README.md [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
+[![Known Vulnerabilities](https://snyk.io/test/github/5app/package-template/badge.svg)](https://snyk.io/test/github/5app/package-template)
 [![CircleCI](https://circleci.com/gh/5app/package-template.svg?style=shield)](https://circleci.com/gh/5app/package-template)
 
 This is a template for starting new JS projects at 5app.
 
-## Getting started
+## What now
 
-Run `npm init` to configure package.json
+1. Run `npm init` to configure [./package.json](./package.json]).
```
Comment (ikhemissi) [inline README.md:10]: Wow, I didn't know using `1.` in all list items will be replaced with the right numbers automatically. Thanks 🙏

**Review by ikhemissi — APPROVED** [review]

**Review by MrSwitch — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (sadams) [reply .circleci/config.yml:47]: Ok, i'll put it back and update the comment based on your comments and if https://github.com/5app/region-sync/pull/1 works (then the next person to do a private repo can confirm either way). (read the bash but can't tell if it's needed.)

**Review by MrSwitch — CHANGES_REQUESTED** [review]
Tests need to pass for renovate maintain dependencies here.

File: package.json (hunk 2 above)
Comment (MrSwitch) [inline package.json:13]: Unfortunately `package-template` still has a requirement to pass tests, to do things like tag it's own releases and update it's own dependencies. Either we revert this change or fudge the CI test runner job.

```suggestion
		"test": "echo 'implement tests' && exit 0"
```

**Review by MrSwitch — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (sadams) [reply package.json:13]: ok will change 😢

**Pull request merged**